src/List.js

```diff
@@ -45,16 +46,27 @@ class List extends React.Component {
         </li>
       )
     })
-
-    return (
-      <div className='list-wrapper'>
-        <h2 className='rating-header'>Movies by Ratings</h2>
-        <button className='sort-button' onClick={() => this.updateSort()}>📈 SORT ALL</button>
-        <ul className='list'>
-          {this.state.clickedSort % 2 == 0 ? list : reSortedList }
```

Comment:
I think it's good we took this out, based on the rubric specs.

I wonder what the convention is for this. Is it okay to then have a more redundant return without logic? 🤔